### PR TITLE
Fix optimize events follow unaligned columns

### PR DESCRIPTION
**Pull request by linkous8**
- Update output Table config to support SetColMinWidth
- Added minimum widths to non-summary columns to ensure alignment of optimize events output

## Type of Change

- [x] Bug Fix
- [ ] New Feature
- [ ] Breaking Change
- [ ] Refactor
- [ ] Documentation
- [ ] Other (please describe)

## Checklist

- [ ] I have read the [contributing guidelines](/CONTRIBUTING.md)
- [ ] Existing issues have been referenced (where applicable)
- [ ] I have verified this change is not present in other open pull requests
- [ ] Functionality is documented
- [ ] All code style checks pass
- [ ] New code contribution is covered by automated tests
- [ ] All new and existing tests pass

### CODE REVIEW

File: output/output.go [hunk 1]
```diff
@@ -299,6 +300,9 @@ func printTable(cmd *cobra.Command, t *Table) {
 		tw.SetHeader(t.Headers)
 	}
 	tw.AppendBulk(t.Lines)
+	for _, tup := range t.ColumnMinWidths {
+		tw.SetColMinWidth(tup[0], tup[1])
```
Comment (pnickolov) [inline output/output.go:304]: @linkous8 , two questions:
1. Wouldn't it be necessary to check if t.ColumnMinWidths is non-nil here?
2. Given the number of table features we had to expose to support the event tail/list, are we stretching the limits of what the common table handler can provide and, instead, use custom table formatting for the events (or even create a separate output helper for this data type). I know this is somewhat subjective but I am suggesting to be considered.

**Review by pnickolov — CHANGES_REQUESTED** [review]
(the nil check is, I think, required to avoid crashing fsoc in other callers of the custom Table extensions)

File: output/output.go (hunk 1 above)
Comment (linkous8) [reply output/output.go:304]: 1. Using range against a nil-slice produces zero iterations without panicking. I verified this via pen-testing and also confirmed its part of the spec:

	> For a nil slice, the number of iterations is 0.

	* https://github.com/golang/go/issues/1713
	* https://go.dev/ref/spec#For_range
	* https://groups.google.com/g/golang-nuts/c/gVgVLQU1FFE
2. All features added to the Table struct are not specific to events and may be of use to others implementing features using table output.

Comment (pnickolov) [reply output/output.go:304]: Agree. Thanks for the links

**Review by pnickolov — APPROVED** [review]

**Pull request merged**